File: 0083-remove-duplicates-from-sorted-list/0083-remove-duplicates-from-sorted-list.cs
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     public int val;
 *     public ListNode next;
 *     public ListNode(int val=0, ListNode next=null) {
 *         this.val = val;
 *         this.next = next;
 *     }
 * }
 */
public class Solution{
    public ListNode DeleteDuplicates(ListNode head){
        ListNode curr=head; // current node ko track kar lo

        while(curr!=null&&curr.next!=null){
            if(curr.val==curr.next.val){
                curr.next=curr.next.next; // duplicate node ko skip kar do
            }
            else{
                curr=curr.next; // next node par chale jao
            }
        }
        return head; // updated linked list return kar do
    }
}

File: 0083-remove-duplicates-from-sorted-list/0083-remove-duplicates-from-sorted-list.go
/**
 * Definition for singly-linked list.
 * type ListNode struct {
 *     Val int
 *     Next *ListNode
 * }
 */
func deleteDuplicates(head *ListNode) *ListNode {
	curr := head // current node ko track kar lo

	for curr != nil && curr.Next != nil {
		if curr.Val == curr.Next.Val {
			curr.Next = curr.Next.Next // duplicate node ko skip kar do
		} else {
			curr = curr.Next // next node par chale jao
		}
	}
	return head // updated linked list return kar do
}